lib/log.h

```diff
@@ -113,6 +113,7 @@ void increaseVerbosity();
     (LOGGING(N) ? ::Log::Detail::fileLogOutput(__FILE__)                  \
                       << ::Log::Detail::OutputLogPrefix(__FILE__, N) << X \
                       << ::Log::Detail::clearPrefix << std::endl          \
+                      << std::flush                                       \
```

Comment:
Doesn't `std::endl` already flush?
https://en.cppreference.com/w/cpp/io/manip/endl

Reply:
https://stackoverflow.com/a/50707073

Reply:
Not sure how that crept in there -- I think at one point I was having an issue with log messages not coming out as I expected and was tweaking things to try to make it work, but that should not have been part of this change